@codex please fix

```
> dspace@3.0.0 format:check
> prettier --config .prettierrc --check "**/*.{js,ts,svelte,json,md}"

Checking formatting...
 ✓ tests/eslintPluginPresence.test.ts (1 test) 5ms
 ✓ frontend/tests/gameStateFlush.test.ts (1 test) 6ms
 ✓ tests/checkoutFetchDepth.test.ts (1 test) 4ms
 ✓ tests/npmTestRunsAllSuites.test.ts (1 test) 9ms
 ✓ tests/astroConfigLogLevel.test.ts (1 test) 6ms
 ✓ tests/prismSvelte.test.ts (1 test) 15ms
 ✓ scripts/tests/dbBenchmarkScript.test.ts (1 test) 5ms
 ✓ tests/progressionBalance.test.ts (1 test) 42ms
 ✓ tests/domException.test.ts (1 test) 4ms
 ✓ frontend/tests/sanity.test.ts (1 test) 4ms
 ✓ tests/indexeddb.server.test.ts (1 test) 264ms
[warn] tests/itemContainers.test.ts
[warn] Code style issues found in the above file. Run Prettier with --write to fix.
 ❯ scripts/tests/prettierFormatting.test.ts (1 test | 1 failed) 17879ms
   × Prettier formatting > frontend sources are formatted 17877ms
     → Command failed: npm run format:check

⎯⎯⎯⎯⎯⎯⎯ Failed Tests 1 ⎯⎯⎯⎯⎯⎯⎯

 FAIL  scripts/tests/prettierFormatting.test.ts > Prettier formatting > frontend sources are formatted
Error: Command failed: npm run format:check
 ❯ scripts/tests/prettierFormatting.test.ts:8:5
      6|   test('frontend sources are formatted', () => {
      7|     const frontendDir = path.join(__dirname, '../../frontend');
      8|     execSync('npm run format:check', {
       |     ^
      9|       cwd: frontendDir,
     10|       stdio: 'inherit',

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[1/1]⎯


 Test Files  1 failed | 289 passed (290)
      Tests  1 failed | 1014 passed (1015)
   Start at  23:15:47
   Duration  153.54s (transform 9.65s, setup 10.20s, collect 43.33s, tests 101.20s, environment 171.68s, prepare 32.52s)


Error: Error: Command failed: npm run format:check
 ❯ scripts/tests/prettierFormatting.test.ts:8:5

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯
Serialized Error: { status: 1, signal: null, output: [ null, null, null ], pid: 9215, stdout: null, stderr: null }

 ELIFECYCLE  Command failed with exit code 1.
Error: Process completed with exit code 1.
```

new codex task, not a r/e/v/i/e/w task

--- a/frontend/tests/itemContainers.test.ts
+++ b/frontend/tests/itemContainers.test.ts
@@ -81,7 +81,6 @@ describe('itemContainers helpers', () => {
         expect(removeAllStoredItems(jarId, dusdId)).toBe(0);
     });
 
-
     test('normalizes malformed persisted counts to non-negative finite values', () => {
         mockGameState.itemContainerCounts[jarId] = {
             [dusdId]: Number.NaN,

